devenv/local/docker-compose/Dockerfile.emily

```diff
@@ -0,0 +1,72 @@
+FROM rust:bookworm as builder
+
+# Install dependencies.
+RUN apt-get update
+RUN apt-get install -y \
+    libclang-dev \
+    git pkg-config \
+    libssl-dev make \
+    protobuf-compiler \
+    npm \
+    default-jre
+RUN npm install -g pnpm@9
+RUN npm install -g @openapitools/openapi-generator-cli
+RUN rustup toolchain install stable
+RUN rustup component add rustfmt --toolchain stable
+# Setup the code.
+RUN mkdir /code
+WORKDIR /code
+RUN git clone https://github.com/stacks-network/sbtc.git
+WORKDIR /code/sbtc
+
+# TODO: DELETE THIS LINE BEFORE MERGING vvv
+RUN git checkout chore/run-signer-binary-against-devenv-emily-as-server
+# TODO: DELETE THIS LINE BEFORE MERGING ^^^
+
+RUN make install
+RUN make build
+
+# Generate CDK template
+# ------------------------------------------------------------------------------
+ARG AWS_STAGE=local
+ARG TABLES_ONLY=true
+RUN pnpm --filter emily-cdk run synth
+
+# NOTE: If you want to synthsize with the lambda and apigateway instance as well
+# you need to also compile the emily handler as a lambda and generate the openapi
+# spec before attempting to synthesize the cdk template.
+
+# Compile emily handler as a warp server.
+# ------------------------------------------------------------------------------
+RUN cargo build --release --bin emily-server
+RUN ls -l ./target/release/emily-server
+
+# Create Docker image to setup AWS resources.
+# ------------------------------------------------------------------------------
+FROM python:3.12-slim AS aws-setup
+WORKDIR /code
+COPY --from=builder /code/sbtc/devenv/aws-setup/initialize.py /code/initialize.py
+COPY --from=builder /code/sbtc/emily/cdk/cdk.out /code/cdk.out
+RUN pip3 install boto3
+# The local lambda path is not used here because we're only deploying with the
+# dynamodb tables. This will need to point to the local lambda zip file
+# that was compiled to be deployed.
+ENV LOCAL_LAMBDA_PATH=/code/your-compiled-aws-lambda-here.zip
+ENV DYNAMODB_ENDPOINT=http://dynamodb:8000
+ENV INPUT_CDK_TEMPLATE=/code/cdk.out/EmilyStack.template.json
+ENV OUTPUT_CDK_TEMPLATE=/code/cdk.out/EmilyStack.devenv.template.json
+ENTRYPOINT ["python3", "/code/initialize.py"]
```

Comment:
Why add this here?

Reply:
And I would think that we need something to run this, or does it happen somehow?

Reply:
It happens in the `docker-compose`, called `emily-aws-setup`.

Reply:
Ahhh, I see. This makes sense.